ui/packages/atlasmap/src/UI/dialogs/PropertyDialog.tsx

```diff
@@ -27,6 +27,7 @@ export interface IProperty {
   name: string;
   value: string;
   valueType: string;
+  initialName: string;
```

Comment:
There is no need for this, the initial data is already passed and available, check my comment in the issue